src/libraries/System.Text.RegularExpressions/tools/GenerateRegexCasingTable.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
       <OutputType>Exe</OutputType>
       <TargetFramework>net7.0</TargetFramework>
+      <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
+      <LangVersion>latest</LangVersion>
```

Comment:
Is LangVersion latest required in this project? This project doesn't even build as part of the root build / source build.

Reply:
No, I don't think so.  Removed.